<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>96</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="ProductFrame">
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color:  rgb(255,255,255);
	border-radius: 2;
	border-color: #ffc107;
	border-width: 2px;

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0,0,0,0">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="ProductNameLabel">
                  <property name="font">
                   <font>
                    <family>Futura</family>
                    <pointsize>20</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>Product Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="pushButton_5">
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
    color: #ffffff;
	background-color: rgb(254, 177, 40);
	border-radius:10;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color: rgb(254, 177, 40);
	background-color: #ffffff;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:disabled{
    color: rgb(254, 177, 40);
    border-width: 1px;
    border-color: rgb(254, 177, 40);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                    </property>
                    <property name="text">
                     <string>Discount 10%</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_4">
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
    color: #ffffff;
	background-color: rgb(55, 219, 27);
	border-radius:10;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color: rgb(55, 219, 27);
	background-color: #ffffff;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:disabled{
    color: rgb(55, 219, 27);
    border-width: 1px;
    border-color: rgb(55, 219, 27);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                    </property>
                    <property name="text">
                     <string>Tidak pakai bawang</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_3">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
    color: #ffffff;
	background-color: rgb(55, 219, 27);
	border-radius:10;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color: rgb(55, 219, 27);
	background-color: #ffffff;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:disabled{
    color: rgb(55, 219, 27);
    border-width: 1px;
    border-color: rgb(55, 219, 27);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                    </property>
                    <property name="text">
                     <string>Pedas</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="decreasePushButton">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Futura</family>
                  <pointsize>24</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
    color: #ffffff;
	background-color:  #ffc107;
    border-radius: 10;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color: rgb(96, 97, 97);
	background-color: #ffffff;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:disabled{
    color: rgb(55, 219, 27);
    border-width: 1px;
    border-color: rgb(55, 219, 27);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                </property>
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="ProductQtyLabel">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Futura</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Qty</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="increasePushButton">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Futura</family>
                  <pointsize>24</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
    color: #ffffff;
	background-color:  #ffc107;
    border-radius: 10;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color: rgb(96, 97, 97);
	background-color: #ffffff;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:disabled{
    color: rgb(55, 219, 27);
    border-width: 1px;
    border-color: rgb(55, 219, 27);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="TotalLabel">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Futura</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Total</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="delProductLinePushButton">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
    color: #ffffff;
	background-color: rgb(252, 40, 9);
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;

}
QPushButton:pressed{
    color: rgb(252, 40, 9);
	background-color: #ffffff;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:disabled{
    color: rgb(252, 40, 9);
    border-width: 1px;
    border-color: rgb(252, 40, 9);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
                </property>
                <property name="text">
                 <string>DEL</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
